app/components/List.jsx

```diff
@@ -86,42 +87,35 @@ const styles = theme => ({
   }
 });
 
+const lastModifiedFormatter = new Intl.DateTimeFormat()
+
 class ListAll extends Component {
 
   constructor(props) {
     super(props);
     const { classes } = props;
     this.classes = classes;
-    this.state = { resumes: [], filteredResumes: [] };
-
-    this.filterList = this.filterList.bind(this);
-  }
+    this.state = { resumes: [], abortFetch: () => {} };
 
-  componentDidMount() {
-    authorizedFetch(`/resumes`)
-      .then(res => res.json())
-      .then(data => {
-        this.setState({ resumes: data, filteredResumes: data });
-      });
+    this.searchResumesWithDebounce = debounce(this.searchResumes.bind(this), 200);
   }
 
-  filterList(event) {
-    if (event.target.value.length < 3) {
-      this.setState({ filteredResumes: this.state.resumes });
-    } else {
-      this.setState({
-        filteredResumes: this.state.resumes.filter(resume => {
-          return JSON.stringify(resume).toLocaleLowerCase().includes(event.target.value.toLocaleLowerCase());
-        })
+  searchResumes(search) {
+    const url = new URL("/resumes", "http://example.com");
+    url.searchParams.set("search", search);
+    this.state.abortFetch();
```

Comment:
Is it ok to call `abort` multiple time on the same `AbortController` ?

Reply:
It is not the same `AbortController`. A new one is created every time `abortableAuthorizedFetch` is called.